### PR TITLE
allow gehen.yml to specify role

### PR DESCRIPTION
**Pull request by ohsabry**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -259,12 +260,24 @@ func main() {
...
 		Region: aws.String("us-east-1"),
 	}))
-	ecsClient := ecs.New(sess)
-	ebClient := eventbridge.New(sess)
+
+	// Connect to ECS and EventBridge APIs
+	var ecsClient *ecs.ECS
+	var ebClient *eventbridge.EventBridge
+
+	if role != nil && role.AccountID != "" && role.Name != "" {
```
Comment (ohsabry) [inline main.go:271]: I could also make config return an error if a role was parsed that had one of AccountID and Name being its zero value, will think more tomorrow my brain is busted rn

File: main.go [hunk 2]
```diff
@@ -259,12 +260,24 @@ func main() {
 		fatal.Exit("gehen.yml must contain at least one service")
 	}
 
-	// Connect to ECS API
 	sess := session.Must(session.NewSession(&aws.Config{
 		Region: aws.String("us-east-1"),
```
Comment (ohsabry) [inline main.go:264]: @cszatmary Its a bit sus that we are hardcoding `us-east-1` - If I take this out does it just infer region based on the aws config in shipit or an env var?

File: config/testdata/gehen.good.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
+role:
```
Comment (ohsabry) [inline config/testdata/gehen.good.yml:1]: TODO: make a test fixture with a gehen file without a `role`

File: main.go (hunk 2 above)
Comment (esiddiqui) [reply main.go:264]: yes this takes highest priority, if you skip it, it comes from the credential chain...

**Review by cszatmary — APPROVED** [review]

**Pull request merged**